src/mcts/params.cc

```diff
@@ -375,17 +412,27 @@ void SearchParams::Populate(OptionsParser* options) {
   options->Add<ChoiceOption>(kHistoryFillId, history_fill_opt) = "fen_only";
   options->Add<FloatOption>(kMovesLeftMaxEffectId, 0.0f, 1.0f) = 0.0345f;
   options->Add<FloatOption>(kMovesLeftThresholdId, 0.0f, 1.0f) = 0.0f;
-  options->Add<FloatOption>(kMovesLeftSlopeId, 0.0f, 1.0f) = 0.0027f;
+  options->Add<FloatOption>(kMovesLeftSlopeId, 0.0f, 1.0f) = 0.00f;
```

Comment:
Yes -- unfortunately, the way our moves left head works relies on the value head not providing conflicting information. When using rather extreme contempt and MLH at the same time, the stronger side will often pick the worst move because it has the lowest M.
I'm aware of this not being ideal, and setting the default slope to 0.00 is mainly for people testing out the PR at defaults. @borg323 suggested a check at the end of `params.cc` which throws a warning/expection when both `mlh-slope` and `wdl-rescale-diff` are different from 0, and/or sets slope to 0 automatically in that case.

Reply:
Maybe instead have something to change the default when contempt != 0?

Reply:
There actually is a solution for turning mlh back on: slightly change the interpretation of mlh-threshold and remove the discontinuity caused by it. We likely should give a warning if contempt is set but mlh-threshold is at the default 0.0.